index.html

```diff
@@ -50,7 +51,143 @@ <h2 class=" role"> I am a Software developer</h2>
     </section>
     
 </div>
+    <div class ="worksection">
+        <section id = "worksectionheader">
+        <h1 id= "worksectionheading"> My Recent Works</h1>
```

Comment:
- [ ] The title (e.g., "My Recent Works") uses a header tag hx. You could use `h2` tag. This is essentially due to SEO reasons which many believe it is better to have only one `h1 tag on a webpage.